extensions/amp-analytics/0.1/test/test-amp-analytics.js

```diff
@@ -1936,16 +1938,19 @@ describes.realWin('amp-analytics', {
       };
     };
 
-    const runResourceTimingTest = function(entries, config, expectedPing) {
-      sandbox.stub(win.performance, 'getEntriesByType').returns([entries]);
-      const analytics = getAnalyticsTag(config);
-      return waitForSendRequest(analytics).then(() => {
+    this.timeout(400);
+
+    const runResourceTimingTest = function(
+      entries, config, expectedPing, timeout = 100) {
+      sandbox.stub(win.performance, 'getEntriesByType').returns(entries);
+      analytics = getAnalyticsTag(config);
```

Comment:
Why changed from const?

Reply:
I made some changes that I reverted in the end, but missed this line.